gazelle/tests/simple_import_subdir/BUILD.out

```diff
@@ -0,0 +1,9 @@
+load("@aspect_rules_ts//ts:defs.bzl", "ts_project")
+
+ts_project(
+    name = "simple_import_subdir",
+    srcs = [
+        "main.ts",
+        "subdir/lib.ts",
```

Comment:
why doesn't this example generate a BUILD file under subdir/ ? Because there's no BUILD.in file?

Reply:
Currently it will never create a BUILD, only modify existing ones.

I think that is what we'd want? From what I've seen+created it's very common to have ts code spread across folders, things like angular components (ts, css, html) each having their own folder and a parent maybe containing the `@NgModule` and BUILD etc.

Is there a common gazelle convention/norm for *creating* BUILD files?

Reply:
Yeah it should work to run gazelle on a project with zero build files. Maybe this is just a quirk of the test runner though? Will try hooking up in a real example.